public_html/lib/auth.php

```diff
@@ -3,16 +3,18 @@
 
  function encryptpwd($pass){
     global $config;
-    $iv_size = mcrypt_get_iv_size(MCRYPT_RIJNDAEL_256, MCRYPT_MODE_ECB);
+    /*$iv_size = mcrypt_get_iv_size(MCRYPT_RIJNDAEL_256, MCRYPT_MODE_ECB);
     $iv      = mcrypt_create_iv($iv_size, MCRYPT_RAND);
-    return trim(base64_encode(mcrypt_encrypt(MCRYPT_RIJNDAEL_256, $config['ckey'], $pass, MCRYPT_MODE_ECB, $iv)));
+    return trim(base64_encode(mcrypt_encrypt(MCRYPT_RIJNDAEL_256, $config['ckey'], $pass, MCRYPT_MODE_ECB, $iv)));*/
+    return trim(base64_encode($pass));
   }
 
   function decryptpwd($pass){
     global $config;
-    $iv_size = mcrypt_get_iv_size(MCRYPT_RIJNDAEL_256, MCRYPT_MODE_ECB);
+    /*$iv_size = mcrypt_get_iv_size(MCRYPT_RIJNDAEL_256, MCRYPT_MODE_ECB);
     $iv      = mcrypt_create_iv($iv_size, MCRYPT_RAND);
-    return trim(mcrypt_decrypt(MCRYPT_RIJNDAEL_256, $config['ckey'], base64_decode($pass), MCRYPT_MODE_ECB, $iv));
+    return trim(mcrypt_decrypt(MCRYPT_RIJNDAEL_256, $config['ckey'], base64_decode($pass), MCRYPT_MODE_ECB, $iv));*/
+    return trim(base64_decode($pass));
   }
```

Comment:
I hope these changes are temporary, because commenting out the login cookie encryption undermines the login cookie security system (that allows only allowed IP ranges) and leaks the password hash.

Reply:
Indeed! temporary and considered 'unstable'. We're about to put the development branch it into an unstable state in order to start converting over to PHP 7.x compliance. Right now MCRYPT support is no longer supported by PHP7. We're going to need to replace it with RIJNDAEL or similar functionality. 

I'm going to make a stable/php5.6 branch or similar for the code that will be safe for live situations. Don't worry this won't be used live until everything is fixed, and we'll be putting warnings on the branch with the php7 changes until it reaches stability.

Reply:
The goal of this set of changes is simply to get the core functionality to be usable on PHP7. As mcrypt is a removed feature of PHP7+, it's removed until a suitable alternative is decided upon.

I'm more than aware this is a less than ideal situation for now. Further updates to come, as issues are found, etc.

Reply:
Please keep suggesting any thing of any issue as we move along this process. I've created a stable branch where the 5.6 compatible only version will live until the 7.x version becomes stable itself.